hubblestack/modules/config.py

```diff
@@ -0,0 +1,373 @@
+# -*- coding: utf-8 -*-
+'''
+Return config information
+'''
+
+# Import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import copy
+import re
+import os
+import logging
+
+# Import salt libs
+import hubblestack.config
+import salt.utils.data
+import salt.utils.dictupdate
+import salt.utils.files
+import salt.utils.platform
+try:
+    # Gated for salt-ssh (salt.utils.cloud imports msgpack)
+    import salt.utils.cloud
+    HAS_CLOUD = True
+except ImportError:
+    HAS_CLOUD = False
+
+import salt._compat
+import salt.syspaths as syspaths
+import salt.utils.sdb as sdb
+
+# Import 3rd-party libs
+from salt.ext import six
```

Comment:
We should probably get rid of this, the `salt.ext` has already been removed everywhere else I think and we'd be stuck with it, and change `six.string_types` to just `str`

Reply:
I thought I got all these. Nice catch.